pkg/driver/node.go

```diff
@@ -169,6 +173,87 @@ func (d *nodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 		return nil, status.Error(codes.InvalidArgument, "Target path not provided")
 	}
 
+	volumeId := req.GetVolumeId()
+	klog.V(5).Infof("NodePublishVolume: volume_id is %s", volumeId)
+
+	// check mount pod is need to delete
+	klog.V(5).Infof("Check mount pod is need to delete or not.")
+	k8sClient, err := juicefs.NewClient()
+	if err != nil {
+		klog.V(5).Infof("Can't get k8s client: %v", err)
+		return &csi.NodeUnpublishVolumeResponse{}, nil
+	}
+
+	pod, err := juicefs.GetPod(k8sClient, juicefs.GetConfigMapNameByVolumeId(volumeId), juicefs.Namespace)
```

Comment:
GetPodNameByVolumeId ?

Reply:
Pod name is formed as <nodeName>-<volumeId> , change the function name to GeneratePodNameByVolumeId